gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-bin.zip
```

Comment:
Did you mean to update this? If so, I would do it via a separate PR with an explanation of why it's being updated.

Reply:
com.github.jk1.dependency-license-report" version "1.3" requires at least grade version 3.3
As discussed, upgraded to gradle wrapper version 4.10.2